pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
+
+`composer self-update`
+
+- Make a package a dependency for this project:
```

Comment:
What defines a project (e.g. "this" project)? Is there a file similar to package.json, or does the command simply assume we mean the current directory?